home-application/src/main/java/com/softserveinc/ita/homeproject/application/api/UserApiImpl.java

```diff
@@ -28,10 +36,11 @@
  */
 @Provider
 @NoArgsConstructor
-public class UserApiImpl implements UsersApi {
+public class UserApiImpl extends CommonApi<UserDto> implements UsersApi {
 
     private UserService userService;
     private HomeMapper mapper;
+    private EntitySpecificationService entitySpecificationService;
```

Comment:
Can we move mapper and entitySpecificationService into CommonApi?

Reply:
If we move  mapper and entitySpecificationService into CommonApi We'll get NullPointerException